plugins/modules/aci_access_switch_policy_group.py

```diff
@@ -0,0 +1,472 @@
+    - The APIC defaults to C("") which results in the target DN set to the default policy when unset during creation.
+    type: str
+  equipment_flash_policy:
+    description:
+    - The equipment flash policy bound to the access switch policy group.
+    - The APIC defaults to C("") which results in the target DN set to the default policy when unset during creation.
+    type: str
+  cdp_policy:
+    description:
+    - The CDP policy bound to the access switch policy group.
+    - The APIC defaults to C("") which results in the target DN set to the default policy when unset during creation.
+    type: str
+  lldp_policy:
+    description:
+    - The LLDP policy bound to the access switch policy group.
+    - The APIC defaults to C("") which results in the target DN set to the default policy when unset during creation.
+    type: str
+  sync_e_node_policy:
+    description:
+    - The SyncE node policy bound to the access switch policy group.
+    - Only available in APIC version 5.x or later.
+    - The APIC defaults to C("") which results in the target DN set to the default policy when unset during creation.
+    type: str
+  ptp_node_policy:
+    description:
+    - The PTP node policy bound to the access switch policy group.
+    - Only available in APIC version 5.2 or later.
+    - The APIC defaults to C("") which results in the target DN set to the default policy when unset during creation.
+    type: str
+  usb_configuration_policy:
+    description:
+    - The USB configuration policy bound to the access switch policy group.
+    - Only available in APIC version 5.2 or later.
+    - The APIC defaults to C("") which results in the target DN set to the default policy when unset during creation.
+    type: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+seealso:
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(infra:AccNodePGrp).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Tim Cragg (@timcragg)
+- Akini Ross (@akinross)
+"""
+
+EXAMPLES = r"""
+- name: Create Access Switch Policy Group
+  cisco.aci.aci_access_switch_policy_group:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    name: ansible_pol_grp
+    spanning_tree_policy: example_spanning_tree_policy
+    bfd_ipv4_policy: example_bfd_ipv4_policy
+    bfd_ipv6_policy: example_bfd_ipv6_policy
+    fibre_channel_node_policy: example_fibre_channel_node_policy
+    poe_node_policy: example_poe_node_policy
+    fibre_channel_san_policy: example_fibre_channel_san_policy
+    monitoring_policy: example_monitoring_policy
+    copp_leaf_policy: example_copp_leaf_policy
```

Comment:
if we do the two classes in one module. Should we rename this copp_policy?

Reply:
I would keep it as copp_leaf_policy, and then have copp_spine_policy added. If decided to do both classes in one module. Also to highlight additionally that these are separate classes and this is the matching the UI.